internal/services/zero_trust_access_application/testdata/accessapplicationconfigwithadefinedidp.tf

```diff
@@ -1,8 +1,8 @@
-
 resource "cloudflare_zero_trust_access_identity_provider" "%[1]s" {
   account_id = "%[4]s"
   name = "%[1]s"
   type = "onetimepin"
+  config = {}
```

Comment:
we shouldn't need to put empty blocks anywhere. this should be handled without it being present.

Reply:
I fixed that test in a second commit that I hadn't pushed up yet. The config block is non-empty in the working test.